skyportal/utils/offset.py

```diff
@@ -478,6 +478,14 @@ def _calculate_best_position_for_offset_stars(
         log("Uncaught error in ra, dec determination")
         return fallback
 
+    if np.isnan(med_ra) or np.isnan(med_dec):
+        log(
+            "Warning: the median of the positions"
+            " from the photometry data associated with this source "
+            " retured nan, using fallback"
+        )
+        return fallback
```

Comment:
This seems unrelated, but I guess it's good to have!

Reply:
yeah just something.i noticed.